src/particles/ImpactXParticleContainer.H

```diff
@@ -155,6 +166,43 @@ namespace impactx
         RefPart const
         GetRefParticle () const;
 
+
+        /** Get reference particle relativistic gamma
+         *
+         * @param refpart reference particle
+         * @returns relativistic gamma
+         */
+        amrex::ParticleReal
+        GetGamma (RefPart const refpart);
```

Comment:
Since we have access to the reference particle, we could make this an empty argument.

Alternatively, we could implement all setters and getters directly in the `RefPart` class?